lib/Target/SubstraitPB/Export.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include <google/protobuf/text_format.h>
 #include <google/protobuf/util/json_util.h>
+#include <mlir/IR/ValueRange.h>
```

Comment:
I was told that MLIR and LLVM (1) always include other LLVM or MLIR headers using `""` and (2) we shouldn't include some of the most basic headers because they are always included transitively anyways (i.e., LLVM does not follow "include what you use"). That line was probably added by VS Code automatically... TL;DR: Remove that line.